react-native/training/src/config/api.js

```diff
@@ -1,7 +1,10 @@
 import Reactotron from 'reactotron-react-native';
 import { create } from 'apisauce';
+import Config from 'react-native-config';
 
-const baseURL = 'http://wolox.com';
+import { isAndroid } from '../constants/platform';
+
+const baseURL = isAndroid ? Config.API_BASE_URL_ANDROID : Config.API_BASE_URL_IOS;
```

Comment:
Why not just one api url?

Reply:
because if I leave the API URL does not recognize the connection in Android
![image](https://user-images.githubusercontent.com/40371687/48205897-0c13cd00-e33b-11e8-8b9e-562ce2cfcaaa.png)